components/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/APIProviderImpl.java

```diff
@@ -503,6 +504,20 @@ public int saveEntitlementPolicyPartial(String policyPartialName, String policyP
         return appMDAO.saveEntitlementPolicyPartial(policyPartialName, policyPartial, isSharedPartial, policyAuthor,
                 policyPartialDesc, tenantId);
     }
+    @Override
+    public int saveBusinessOwner(String ownerName, String ownerMail, String description, String sitelink, String keys,
+                                 String values) throws AppManagementException{
+        return appMDAO.saveBusinessOwner(ownerName, ownerMail, description, sitelink, keys, values);
+
+    }
+
+    @Override
+    public int updateBusinessOwner(String ownerId, String ownerName, String ownerMail, String description,
+                                   String sitelink, String keys,
```

Comment:
Pass business owner object instead of business owner properties in 
- saveBusinessOwner
- updateBusinessOwner